ruby/vampires.rb

```diff
@@ -0,0 +1,64 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+until employees == 0
+
+	puts "What is your name?"
+	name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	year_born = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you (y/n)?"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the company's health insurance (y/n)?"
+	insurance = gets.chomp
+
+	puts "What allergies do you have (type 'done' when finished)?"
+	allergies = gets.chomp
+
+	real_age = 2017 - year_born
```

Comment:
Check out the Ruby Time or Date modules in the docs! Instead of hardcoding your year you can use `Date.now.year` or `Time.now.year` to dynamically change the year.